.github/workflows/post-release.yml

```diff
@@ -0,0 +1,57 @@
+name: Post-release version bump
+
+# how to trigger: https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow
+on:
+  workflow_dispatch:
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  ci:
+    if: github.repository == 'blue-build/cli'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install cargo-release
+        run: cargo install cargo-release
+
+      - name: Git setup
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+
+      - name: Setup post-release version bump
+        run: |
+          # Read the current version from Cargo.toml
+          current_version=$(cargo metadata --format-version 1 --no-deps | \
+            jq --raw-output '.packages | .[] | select(.name == "blue-build").version')
+
+          # Sanity check: current version should be 0.X.Y
+          if ! grep -q '^0\.[0-9]\+\.[0-9]\+$' <<< "${current_version}"; then
+            echo "Invalid version (not in 0.X.Y format): ${current_version}"
+            exit 1
+          fi
+
+          minor_version=$(sed 's/^0\.\([0-9]\+\).*/\1/' <<< "${current_version}")
+          next_version=0.$((minor_version + 1)).0-dev
```

Comment:
Just a note, we'll need to update this later to account for the major version too as this seems to only be concerned about versions `0.x.x`

Reply:
noted. i think for bevy they havent release a 1.0.0 version yet so